### PR TITLE
Features branch

### PR DESCRIPTION
**Pull request by sibomana123**
In this Portfolio project, I recall all my past knowledge that I learned in the few days that passed, and it was exciting to work with the Git flow branch

> Added and config webpack, linters
>used ES6 best practice
>used flexbox in style
>To extract data from the Figma design template

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        <li class="social__media__link"><a href="#twitter"><img class="social__media__img" src="img/Linkedin icon (1).svg" alt=""></a></li>
+        <li class="social__media__link"><a href="#twitter"><img class="social__media__img" src="img/medium (2).svg" alt=""></a></li>
+        <li class="social__media__link"><a href="#twitter"><img class="social__media__img" src="img/github (2).svg" alt=""></a></li>
+        <li class="social__media__link"><a href="#twitter"><img class="social__media__img" src="img/angellist.svg" alt=""></a></li>  
+      </ul>
+    </div>  
+      <img class="backg__image" src="img/Header bg.svg" alt="">
+      <img class="backg__image__phone" src="img/header-shapes mobile@2x.svg" alt="">
+  </header>
+
```
Comment (bobb-Rob) [inline index.html]: - [ ] Based on the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md), you are expected to implement the mobile view of this portfolio site and not the desktop view. 

image

Kindly implement this.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      <ul class="social__media">
+        <li class="social__media__link"><a href="#twitter"><img class="social__media__img" src="img/twitter (1).svg" alt=""></a></li>
+        <li class="social__media__link"><a href="#twitter"><img class="social__media__img" src="img/Linkedin icon (1).svg" alt=""></a></li>
+        <li class="social__media__link"><a href="#twitter"><img class="social__media__img" src="img/medium (2).svg" alt=""></a></li>
+        <li class="social__media__link"><a href="#twitter"><img class="social__media__img" src="img/github (2).svg" alt=""></a></li>
+        <li class="social__media__link"><a href="#twitter"><img class="social__media__img" src="img/angellist.svg" alt=""></a></li>  
+      </ul>
+    </div>  
+      <img class="backg__image" src="img/Header bg.svg" alt="">
+      <img class="backg__image__phone" src="img/header-shapes mobile@2x.svg" alt="">
```
Comment (bobb-Rob) [inline index.html]: - [ ] The headline section has the proper background image (decorations are always background images, not `img` tags). If an image is not part of the content (for aesthetic reasons) use a CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info. In other words, The background image should not `` tag but should be added using CSS background property.  Kindly fix this.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  </header>
```
Comment (bobb-Rob) [inline index.html]: - [ ] The container of the headline section should be a `section` tag. It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info. This means that your `<header>` should contain only the logo and the `nav` hamburger menu element. The the `<div class="description">`  should be a section tag. Like so 👇🏾 


```html
<header>
   <logo>
   <nav>
</header>
<section>
   <h1>
    other elements.
</section>
```

File: index.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+      
+      <ul class="social__media">
+        <li class="social__media__link"><a href="#twitter"><img class="social__media__img" src="img/twitter (1).svg" alt=""></a></li>
+        <li class="social__media__link"><a href="#twitter"><img class="social__media__img" src="img/Linkedin icon (1).svg" alt=""></a></li>
+        <li class="social__media__link"><a href="#twitter"><img class="social__media__img" src="img/medium (2).svg" alt=""></a></li>
+        <li class="social__media__link"><a href="#twitter"><img class="social__media__img" src="img/github (2).svg" alt=""></a></li>
+        <li class="social__media__link"><a href="#twitter"><img class="social__media__img" src="img/angellist.svg" alt=""></a></li>  
+      </ul>
+    </div>  
+      <img class="backg__image" src="img/Header bg.svg" alt="">
```
Comment (bobb-Rob) [inline index.html]: - [x] This is how your project looks in the mobile view screens size of `375px`. Kindly fix this, and make sure that your implementation looks exactly like the Figma design for mobile. 

image

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @sibomana123, 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi  @sibomana123 👋

Your project is complete! ✔️✔ , I loved your work, it looks great 🔥🔥🔥🔥 🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors

- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**